domain/schema/model/sql/0019-application.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE application (
     password_hash_algorithm_id TEXT,
     password_hash TEXT,
     -- space_uuid is the default binding for this application.
-    space_uuid TEXT,
+    space_uuid TEXT NOT NULL,
```

Comment:
I know you didn't add this column, but isn't it redundant with the `space_uuid` in `application_endpoint`? I don't see the point of storing the space we bound to at deploy time

Reply:
It's the default space. See: https://github.com/juju/juju/pull/19240#discussion_r2003964757

Reply:
Ahh right @SimonRichardson, I was certain I was missing some context.